srcpkgs/ansible/template

```diff
@@ -1,14 +1,14 @@
 # Template file for 'ansible'
 pkgname=ansible
-version=2.10.1
+version=3.3.0
```

Comment:
Is this official? I can't find it anywhere but pypi.

Reply:
yep, this is official. See https://docs.ansible.com/ansible/latest/roadmap/COLLECTIONS_3_0.html and https://www.ansible.com/blog/ansible-3.0.0-qa. They split out all the modules, plugins etc into "collections", and renamed the rest to ansible-base. The new ansible package is a package where they bundle those collections,  whose content was included in the old ansible. While those two packages both started at 2.10, the version numbers are now diverging. Ansible 3.0 is based on ansible-base 2.10, Ansible 4.0 will be based on ansible-base 2.11, and so on.